packages/react/src/components/Badge/index.test.tsx

```diff
@@ -0,0 +1,46 @@
+import React from 'react';
+import { render, screen } from '@testing-library/react';
+import Badge, { BadgeLabel } from '../../../src/components/Badge';
+import axe from '../../axe';
+
+test('renders children', () => {
+  render(
+    <Badge>
+      <BadgeLabel>Label:</BadgeLabel> value
+    </Badge>
+  );
+  expect(screen.getByText('Label:')).toBeInTheDocument();
+});
+
+test('passes classNames through', () => {
+  render(
+    <Badge className="baz">
+      <BadgeLabel className="jazz">Badge:</BadgeLabel>
+      hi
+    </Badge>
+  );
+  expect(screen.getByText('hi')).toHaveClass('baz');
+  expect(screen.getByText('Badge:')).toHaveClass('jazz');
+});
+
+test('passes arbitrary props through', () => {
+  render(
+    <Badge data-foo="true">
+      <BadgeLabel data-bar="yes">hi</BadgeLabel>
+      bye
+    </Badge>
+  );
+  expect(screen.getByText('bye')).toHaveAttribute('data-foo', 'true');
+  expect(screen.getByText('hi')).toHaveAttribute('data-bar', 'yes');
+});
+
+test('should return no axe violations', async () => {
```

Comment:
Per our guidelines we need to include all variants as part of testing, including the "small" variant, see https://github.com/dequelabs/cauldron/blob/develop/CONTRIBUTING.md#testing-strategies

Reply:
These guidelines also include accessibility testing, so a test for `size="small"` is needed as well.